en-US/firefox/facebookcontainer/index.lang

```diff
@@ -0,0 +1,73 @@
+## NOTE: Demo page at https://www-dev.allizom.org/firefox/facebookcontainer/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/facebookcontainer/
+
+
+# HTML page title
+;Facebook Container for Firefox | Prevent Facebook from seeing what websites you visit.
+Facebook Container for Firefox | Prevent Facebook from seeing what websites you visit.
+
+
+# HTML page description
+;Millions of people around the world trust Firefox Web browsers on Android, iOS and desktop computers. Fast. Private. Download now!
+Millions of people around the world trust Firefox Web browsers on Android, iOS and desktop computers. Fast. Private. Download now!
+
+
+;Facebook. Well contained. Keep the rest of your life to yourself.
+Facebook. Well contained. Keep the rest of your life to yourself.
+
+
+;Download Firefox and get the Facebook Container Extension
+Download Firefox and get the Facebook Container Extension
+
+
+# The link points to https://addons.mozilla.org/firefox/addon/facebook-container/
+;Get the Facebook Container Extension
+Get the Facebook Container Extension
+
+
+;The Facebook Container Extension is not available on mobile devices.
+The Facebook Container Extension is not available on mobile devices.
+
+
+# GLink points to https://www.mozilla.org/firefox/focus/; Product name in German is Firefox Klar.
+;Try <strong>Firefox Focus</strong>, the privacy browser for Android and iOS.
+Try <strong>Firefox Focus</strong>, the privacy browser for Android and iOS.
+
+
+;Download on the App Store
+Download on the App Store
+
+
+# The video contains text in English
+;Watch the video
+Watch the video
+
+
+;Opt out on your terms
+Opt out on your terms
+
+
+# The link points to https://addons.mozilla.org/firefox/addon/facebook-container/
+;Facebook can track almost all your web activity and tie it to your Facebook identity. If that’s too much for you, the <a href="%(fbcontainer)s">Facebook Container extension</a> isolates your identity into a separate container tab, making it harder for Facebook to track you on the web outside of Facebook.
+Facebook can track almost all your web activity and tie it to your Bacebook identity. If that’s too much for you, the <a href="%(fbcontainer)s">Facebook Container extension</a> isolates your identity into a separate container tab, making it harder for Facebook to track you on the web outside of Facebook.
```

Comment:
Notice anything wrong? :-\

Also, you need for a developer to fix the original string before exposing it, and push it to production before the weekend. Have you checked with Craig? It shouldn't be a problem, but it need to be confirmed.

Reply:
I see there's a PR, that's good. Still, there's a typo in your string.